eq-author-api/utils/defaultMetadata.js

```diff
@@ -93,8 +95,54 @@ const defaultValues = [
   },
 ];
 
+const defaultBusinessSurveyMetadata = [
```

Comment:
Could this be a subset of the above list? Instead of maintaining two default values etc. IIRC during the planning the values (the only difference) were just examples and did not have to be followed.

Reply:
Good idea 👍